ngraph/core/src/rt_info.cpp

```diff
@@ -59,3 +59,34 @@ void ngraph::copy_runtime_info(const ngraph::NodeVector& from, ngraph::NodeVecto
         rtInfoTo = mergedInfo;
     }
 }
+
+static void merge_runtime_info(const ngraph::Node::RTMap& rt_info, std::shared_ptr<ngraph::Node> to) {
```

Comment:
Why do we need these methods?

Reply:
Without this function we can lose some values from the RT info map. The copy_runtime_info is simply overwrite all RT info.